### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by hazwanizafirah**

### CODE REVIEW

**Comment by hazwanizafirah** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,56 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.7.0'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails', branch: 'main'
+gem 'rails', '~> 6.1.4', '>= 6.1.4.1'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3', '~> 1.4'
```
Comment (anlvq-1058) [inline Gemfile:9]: please use mysql2 instead

File: config/credentials.yml.enc [hunk 2]
```diff
@@ -0,0 +1 @@
+XtA7ZjEX3SBbaR17Bfai3+LDSKLZmp2lCYrYcgI2zvUjQyBZerWq64bZed24+8R5V9/Yr2+cfIn2FUVmUNEK+rlCLxoluYOqgdYxYva5l61QpEY72uJPlr51uyIzMC4JMivtEHGCLjJNLlNniWFqNevm8O9ZBbZ1ghIB2AXiQQwbD1tPsHYrOHJl6i4iX2SeDShbocYcTGv35LHPbPMQzWWVRQhC02KjQ3wHm4IPWaSo3n1eadZqoWgriDjCkdAtJgG1ztKvLFf1x9OOAmy5Sb1ebmzj2NM4IvYtZgCmDWk5C59yNv5JEMC5vhaR2J2jGq/Dh42wfnMSwOJe1OuFZn7nh9IzsliRmnZs/THn3U1FXexqXUk79l5O7vJk5Vg2C8tk4he0MCGAQe91ctxcHkhkwNqRd/oJm+Cf--aqYk0Xya0gPFmsRH--WyO22Gw1ku6EwJxqQJf1AA==
```
Comment (anlvq-1058) [inline config/credentials.yml.enc:1]: Please use git ignore to ignore this file

File: config/database.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
```
Comment (anlvq-1058) [inline config/database.yml:7]: Please use git ignore to ignore this file. Them create new file name database.yml.example

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Pull request closed without merging**